khmer/_khmer.cc

```diff
@@ -109,6 +109,32 @@ extern "C" {
 
 /***********************************************************************/
 
+// Convert a hash to a python long object.
+static bool convert_HashIntoType_to_PyObject(const HashIntoType &hashval,
+                                             PyObject **value) {
```

Comment:
Dilemma: would like to keep the interface similar to the other `convert_*` functions but doing this `PyObject **value` and `convert_HashIntoType_to_PyObject(h, &ptr_to_pyobj)` double take seems extremely ugly. Any better ideas?